web/base_viewer.js

```diff
@@ -150,9 +150,9 @@ class BaseViewer {
     ) {
       if (
         !(
-          this.container instanceof HTMLDivElement &&
-          this.viewer instanceof HTMLDivElement
-        )
+          this.container && this.container.tagName.toUpperCase() === "DIV") &&
```

Comment:
The Travis CI build fails because parentheses are missing at the start of this line and the next line. Moreover, there is extra space on line 155. Please fix that and check if the final patch runs correctly.

Reply:
Doh! Assumed a suggested change was safe... should have checked it locally.